website/docs/developer-docs/setup/debugging.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Debugging authentik
+---
+
+This page describes how to debug different parts of an authentik instance, running either in production or in a development setup.
+
+## authentik Server & Worker - Python
+
+The majority of the authentik codebase is in Python, running in gunicorn for the server and celery for the worker. These instructions show how this code can be debugged/inspected.
+
+Note that authentik uses [debugpy](https://github.com/microsoft/debugpy), which relies on the "Debug Adapter Protocol". These instructions demonstrate debugging using [Visual Studio Code](https://code.visualstudio.com/), however they should be adaptable to other editors which support DAP.
+
+To enable the debugging server, set the environment variable `AUTHENTIK_DEBUGGER` to `true`. This will launch the debugging server (by default on port _9901_).
```

Comment:
What do we mean by "the debugging server"? Is that server part of DAP, or do you mean our server, geting debugged by DAP?

Also let's link to `/install-config/configuration/#set-your-environment-variables`.